src/installer/pkg/THIRD-PARTY-NOTICES.TXT

```diff
@@ -0,0 +1,1320 @@
+To anyone who acknowledges that this file is provided "AS IS"
+without any express or implied warranty: permission to use, copy,
+modify, and distribute this file for any purpose is hereby
+granted without fee, provided that the above copyright notices and
+this notice appears in all source code copies, and that none of
+the names of Open Software Foundation, Inc., Hewlett-Packard
+Company, Microsoft, or Digital Equipment Corporation be used in
+advertising or publicity pertaining to distribution of the software
+without specific, written prior permission. Neither Open Software
+Foundation, Inc., Hewlett-Packard Company, Microsoft, nor Digital
+Equipment Corporation makes any representations about the
+suitability of this software for any purpose."
+
+License notice for The LLVM Compiler Infrastructure
+---------------------------------------------------
+
+Developed by:
+
+    LLVM Team
+
+    University of Illinois at Urbana-Champaign
+
+    http://llvm.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal with
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+of the Software, and to permit persons to whom the Software is furnished to do
+so, subject to the following conditions:
+
+    * Redistributions of source code must retain the above copyright notice,
+      this list of conditions and the following disclaimers.
+
+    * Redistributions in binary form must reproduce the above copyright notice,
+      this list of conditions and the following disclaimers in the
+      documentation and/or other materials provided with the distribution.
+
+    * Neither the names of the LLVM Team, University of Illinois at
+      Urbana-Champaign, nor the names of its contributors may be used to
+      endorse or promote products derived from this Software without specific
+      prior written permission.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+CONTRIBUTORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS WITH THE
+SOFTWARE.
+
+License notice for Bob Jenkins
+------------------------------
+
+By Bob Jenkins, 1996.  bob_jenkins@burtleburtle.net.  You may use this
+code any way you wish, private, educational, or commercial.  It's free.
+
+License notice for Greg Parker
+------------------------------
+
+Greg Parker     gparker@cs.stanford.edu     December 2000
+This code is in the public domain and may be copied or modified without 
+permission. 
+
+License notice for libunwind8 based code
```

Comment:
```suggestion
License notice for libunwind based code
```

nit: this is libunwind as in: https://github.com/libunwind/libunwind and their License is MIT which looks like https://github.com/libunwind/libunwind/blob/master/LICENSE (8 is/was just a package version on Ubuntu). We are using copy of this product (as is). See e.g. https://github.com/dotnet/runtime/pull/36988.

Reply:
Thanks @am11 - merging this suggestion.

Reply:
Will the tool add a `libunwind8` notice back into this file if we run it again, since the section header doesn't match anymore? I think it needs to be addressed in both places.

Reply:
> Will the tool add a `libunwind8` notice back into this file if we run it again, since the section header doesn't match anymore? I think it needs to be addressed in both places.

Good catch - I've updated the root TPN file and added to this PR.

Reply:
@NikolaMilosavljevic, thanks for the update. Should the license also be updated to the new (MIT) license? currently we have the old (HP's) license of this library.

Reply:
@am11 would it be fine if we replaced it with the content of this file? https://github.com/libunwind/libunwind/blob/master/LICENSE

Reply:
That is correct. As we are precisely using this revision: https://github.com/libunwind/libunwind/blob/v1.5-rc2/LICENSE here https://github.com/dotnet/runtime/blob/master/src/coreclr/src/pal/src/libunwind/LICENSE.

Reply:
> That is correct. As we are precisely using this revision: https://github.com/libunwind/libunwind/blob/v1.5-rc2/LICENSE here https://github.com/dotnet/runtime/blob/master/src/coreclr/src/pal/src/libunwind/LICENSE.

Thanks - I've pushed the update.